sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -881,14 +881,10 @@ final class DataFrameWriter[T] private[sql](ds: Dataset[T]) {
    *   format("orc").save(path)
    * }}}
    *
-   * You can set the following ORC-specific option(s) for writing ORC files:
-   * <ul>
-   * <li>`compression` (default is the value specified in `spark.sql.orc.compression.codec`):
-   * compression codec to use when saving to file. This can be one of the known case-insensitive
-   * shorten names(`none`, `snappy`, `zlib`, `lzo`, and `zstd`). This will override
-   * `orc.compress` and `spark.sql.orc.compression.codec`. If `orc.compress` is given,
-   * it overrides `spark.sql.orc.compression.codec`.</li>
-   * </ul>
+   * ORC-specific option(s) for writing ORC files can be found in
+   * <a href=
+   *   "https://spark.apache.org/docs/latest/sql-data-sources-orc.html#data-source-option">
+   *   Data Source Option</a> in the version you use.
```

Comment:
Here you didn't link generic options but you did for Parquet. What's diff?

Reply:
This is because the options I removed doesn't include generic options.

Would it be better to just add the link to the Generic options no matter what we removed ??

Reply:
Either way is fine. The point is that we should keep it consistent

Reply:
I would just add it for simplicity, and in case we have write generic options in the future